CONTRIBUTING.md

```diff
@@ -1,65 +1,70 @@
+- Follow the [Quickstart](https://github.com/micromdm/micromdm/wiki/Quickstart) guide and make edits if something doesn't look or work right. 
+- If you run into a problem that you're not sure how to fix, file a bug in [the issue tracker](https://github.com/micromdm/micromdm/issues)
+- Browse through the open issues in [the issue tracker](https://github.com/micromdm/micromdm/issues). We try to tag issues as [**beginner friendly**](https://github.com/micromdm/micromdm/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner-friendly) where appropriate.
+- See something that others might benefit from? Considering updating or writing a [wiki page](https://github.com/micromdm/micromdm/wiki).
 
-# Building the project
+## Building the project
 
-To build MicroMDM from source, you will need [Go 1.8](https://golang.org/dl/) or later installed. 
+To build MicroMDM from source, you will need [Go 1.8](https://golang.org/dl/) or later installed.
 
-## If you have Go
+## Once you have Go
 
 MicroMDM uses the go lang `dep` tool for vendor management. 
 Use `which dep` to verify you have it installed and in your PATH.
 If `dep` is not installed please review the [If you're new to Go](#if-youre-new-to-go) section for install steps.
 
 1. `go get github.com/micromdm/micromdm`
 2. `cd $GOPATH/src/github.com/micromdm/micromdm`
-3. `dep ensure` install the necessary dependencies into /vendor folder
+3. `dep ensure` install the necessary dependencies into  the `/vendor` folder
 4. `go build` or `go install`
 
 ## If you're new to Go
 
-Go is a bit different from other languages in its requirements for how it expects its programmers to organize Go code on a system.
-First, Go expects you to choose a folder, called a workspace (you can name it anything you'd like). The path to this folder must always be set in an environment variable - `GOPATH` (example: `GOPATH=/Users/groob/code/go`)  
-Your `GOPATH` must have thee subfolders - `bin`, `pkg` and `src`, and any code you create must live inside the `src` folder. It's also helpful to add `$GOPATH/bin` to your environment's `PATH` as that is where `go install` will place go binaries that you build.
+Go is a bit different from other languages in its requirements for how it expects its programmers to organize Go code files in directories.
+First, Go expects you to choose a folder, called a workspace (you can name it anything you'd like). The path to this folder must always be set in the environment variable `GOPATH` (example: `export GOPATH=/Users/groob/code/go`).
+Your `GOPATH` must have thee subfolders: `bin`, `pkg`, and `src`. Any code you create must live inside the `src` folder. It's also helpful to add `$GOPATH/bin` to your environment's `PATH` as that is where `go install` will place go binaries that you build. This makes it so that binaries that are insalled can just be invoked by name rathan their full page.
 
 Note: As of Go 1.8 the default `GOPATH` is set to `$HOME/go`.
```

Comment:
Er.. which specific note? The note about bin/pkg/src already comes after the GOPATH setting?